cpp/harvest/nacjd.cc

```diff
@@ -343,9 +355,14 @@ void ParseJSONAndWriteMARC(MARC::Writer * const title_writer) {
 
 
 int Main(int argc, char *argv[]) {
-    if (argc != 3)
+    if (argc < 3 or argc > 4)
```

Comment:
Whenever you change the processing of argc/argv, you should always add documentation in the Usage() output at the top of this file (beginning at line 44).